### DISCUSSION TITLE
I am facing below issue in LingFeast library when I execute "from lingfeat import extractor"

### DISCUSSION THREAD

**Discussion by sarangs-ntnu** [explosion/spaCy#13005] [Help: Coding & Implementations] [answered]
Downloading: https://github.com/yzhangcs/parser/releases/download/v1.1.0/ptb.crf.con.lstm.char.zip to /root/.cache/supar/ptb.crf.con.lstm.char.zip
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
[<ipython-input-8-2b081a97af28>](https://localhost:8080/#) in <cell line: 1>()
----> 1 import lingfeat

4 frames
[/usr/local/lib/python3.10/dist-packages/torch/hub.py](https://localhost:8080/#) in download_url_to_file(url, dst, hash_prefix, progress)
    628         if hash_prefix is not None:
    629             sha256 = hashlib.sha256()
--> 630         with tqdm(total=file_size, disable=not progress,
    631                   unit='B', unit_scale=True, unit_divisor=1024) as pbar:
    632             while True:

TypeError: nop() missing 1 required positional argument: 'it'

**Comment by shadeMe** [comment, marked as answer]
Seems like the issue you're encountering is not related to spaCy. Please use the discussion forum of the `lingfeat` library instead.
